conformance/tests/dataclasses_usage.py

```diff
@@ -172,8 +173,12 @@ class DC13:
     x: int
     x_squared: int
 
+    def __init__(self, x: int, x_squared: int):
```

Comment:
Introducing an `__init__` method here breaks the test. The purpose of this test is to verify that no `__init__` method is synthesized because of the `init=False` argument. This `__init__` should be removed.